var/spack/repos/builtin/packages/r/package.py

```diff
@@ -19,7 +19,8 @@ class R(AutotoolsPackage):
     url = "https://cloud.r-project.org/src/base/R-3/R-3.4.3.tar.gz"
 
     extendable = True
-
+    
```

Comment:
`spack flake8` is showing white space here: the line should be empty, sorry.

Reply:
I removed the whitespace. Let me know if there is anything else :)